### PR TITLE
Ebs

### PR DESCRIPTION
**Pull request by serge-gaia**
lastest EBS tweaks:
* disable the Certbot stuff
* the config.yml was not in the repo.

### CODE REVIEW

File: .ebextensions/04-cerbot.config [hunk 1]
```diff
@@ -0,0 +1,10 @@
+packages:
+  yum:
+    # mod_ssl for apache
+    mod24_ssl.x86_64: []
+
+container_commands:
+  10_installCertbot:
+    command: "wget https://dl.eff.org/certbot-auto;chmod a+x certbot-auto"
+#  20_getCert:
+#    command: "sudo ./certbot-auto certonly --debug --non-interactive --email ${CERT_EMAIL}  --domains ${CERT_DOMAIN} --agree-tos"
```
Comment (tony-gaia) [inline .ebextensions/04-cerbot.config]: It it possible to, rather than disable this command, have it as an option somehow?

Comment (serge-gaia) [reply .ebextensions/04-cerbot.config]: I don't know how. I should delete the whole cerbot thing. We don't need it anymore with the AWS certificate manager.

Comment (tony-gaia) [reply .ebextensions/04-cerbot.config]: Yeah, let's just delete it for now. Can always find it again in the history if we need to re-insert.

**Comment by serge-gaia** [comment]
Removed the cerbot stuff
Also renamed some env variables

**Pull request merged**